src/ucp/core/ucp_ep.h

```diff
@@ -233,61 +246,61 @@ struct ucp_ep_config {
     /* MD index of each lane */
     ucp_md_index_t          md_index[UCP_MAX_LANES];
 
+    struct {
+        /* Maximal total size of rndv_get_zcopy */
+        size_t            max_get_zcopy;
+        /* Minimal size of rndv_get_zcopy */
+        size_t            min_get_zcopy;
+        /* Can the message > `max_get_zcopy` be split to
+         * the segments that are >= `min_get_zcopy` */
+        int               get_zcopy_split;
+        /* Maximal total size of rndv_put_zcopy */
+        size_t            max_put_zcopy;
+        /* Minimal size of rndv_put_zcopy */
+        size_t            min_put_zcopy;
+        /* Can the message > `max_put_zcopy` be split to
+         * the segments that are >= `min_put_zcopy` */
+        int               put_zcopy_split;
+        /* Threshold for switching from eager to RMA based rendezvous */
+        ucp_rndv_thresh_t rma_thresh;
```

Comment:
will rma_thresh/am_thresh from `ucp_ep_config::rndv` used in the future?
currently, I see that they are only set and their values are assigned to `ucp_ep_config::tag::rndv`'s `rma_thresh` and `am_thresh`

Reply:
yes, they will be used by  UCP AM (and maybe stream)
we need separate thresholds for TAG API, because in case of tag offload tag lane may be different than AM lane